kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -386,6 +374,10 @@
         message: 'Showing all available libraries around you.',
         context: 'Connection state when the device is connected and shows other libraries',
       },
+      refresh: {
+        message: 'Refresh',
+        context: 'Link for refreshing library',
+      },
```

Comment:
I think adding/moving around translations strings has been suspended for now due to the string freeze. cc @marcellamaki  @radinamatic

Reply:
I think strings were already in. Might have come due to a rebase yesterday

Reply:
I just checked and the refresh string exists already in the `commonCoreStrings.js`. So using this instead and deleting the new one would be an acceptable solution to this.